src/Http/Event/UpdateSubEventsStatusRequestHandler.php

```diff
@@ -32,7 +32,7 @@ public function __construct(
         $this->validator = $validator;
     }
 
-    public function __invoke(Request $request, string $eventId)
+    public function handle(Request $request, string $eventId)
```

Comment:
It was worth a try :D

Reply:
Yeah I'd also have preferred this since it was a lot cleaner in the controller provider in Silex, but I got an error along the lines of `0 arguments passed to UpdateSubEventsStatusRequestHandler, 2 expected`. So I think Silex was trying to instantiate the class itself and failing to do so correctly.